attributes/default.rb

```diff
@@ -39,3 +39,9 @@
 }
 
 default['iptables']['system_ruby'] = '/usr/bin/ruby'
+case node['platform_family']
+when 'rhel', 'fedora', 'amazon'
+  default['iptables']['persisted_rules'] = '/etc/sysconfig/iptables'
```

Comment:
Since this isn't something the user would ever change we should just use a variable in the recipe instead of an attribute. In the past we used attributes as variables a lot and we're trying to move away from that so that it's super clear to users what they should be tuning using attributes.

Reply:
Hi @tas50 ,
I understand your point but good thing with an attribute is it can be used elsewhere on the cookbook. I'm preparing a new PR to improve idempotence between active rules and rules declared with chef and i use this attribute to compare files. (you can have a look to my branch check_consistency_sysconfig on my fork)